command/plugin_reload_status.go

```diff
@@ -75,7 +76,13 @@ func (c *PluginReloadStatusCommand) Run(args []string) int {
 	})
 
 	if err != nil {
-		c.UI.Error(fmt.Sprintf("Error retrieving plugin reload status: %s", err))
+		if w, ok := err.(errwrap.Wrapper); ok {
```

Comment:
Just curious, what was the motivation behind this change? Is the error typically wrapped? Why the need to explicitly check and unwrap?

Reply:
The error we want to get at, that it's an enterprise feature, is the inner not outer error.  If you call .Error() on a wrapped error you only get the outer message.